### PR TITLE
Election followup

### PR DESCRIPTION
**Pull request by Hoobie**

### CODE REVIEW

File: server/src/main/java/pl/edu/agh/RaftServer.java [hunk 1]
```diff
@@ -311,7 +318,6 @@ private void sendMessageToNeighbors() {
         RaftMessage message = (messagesToNeighbors.size() > 0) ? messagesToNeighbors.remove() : new AppendEntries(currentTerm);
 
         serverConnections.forEach((remoteAddress, connection) -> {
-            votedFor = localAddress;
```
Comment (Hoobie) [inline server/src/main/java/pl/edu/agh/RaftServer.java:314]: I've found your copy-paste mistake ;)

**Comment by AndrzejRyl** [comment]
Still doesn't work ;) try to start 3 servers and kill the leader. Two other ones will just restart election over and over again

**Comment by Hoobie** [comment]
This branch finishes the election part of the  algorithm. Debugging of communication will be performed later.

**Review by AndrzejRyl — COMMENTED** [review]
Okkk... let's do it like that xD

**Review by AndrzejRyl — APPROVED** [review]

**Pull request merged**